Modal/example.py

```diff
@@ -2,12 +2,12 @@
 import os
 import chain
 
-# create an image and imo=port needed dependencies
-image = modal.Image.debian_slim().pip_install("langchain").pip_install("openai").pip_install("python-dotenv")
-stub = modal.Stub()
+# create an image and install needed dependencies
+image = modal.Image.debian_slim().pip_install("langchain", "openai", "python-dotenv")
+stub = modal.Stub(image=image)
```

Comment:
if using one image object in an app, it's easiest to set it as default on the stub.